logger/logger.go

```diff
@@ -0,0 +1,19 @@
+package logger
+
+import (
+	"github.com/rs/zerolog"
+	"os"
+	"sync"
+)
+
+var once sync.Once
+
+var log zerolog.Logger
+
+func Get() zerolog.Logger {
```

Comment:
The logger should be configured and provided by the client. IMO here we can implement a void/null logger which does nothing.

I'm not sure about this global/static getter but it's not a blocker from my side

Reply:
I understand but it's not a web app, it's lib ;-)